### PR TITLE
Remove dhs from scans

### PR DESCRIPTION
**Pull request by jsf9k**

### CODE REVIEW

**Review by KyleEvers — COMMENTED** [review]
Why dashes and underscores in the csv name? Is that correct?

**Comment by jsf9k** [comment]
It is correct, but admittedly odd.  The original CSV that I download from another source is called `current-federal.csv`.  I'm modifying that one and creating `current-federal_modified.csv`.  I'm using the `-` and `_` to differentiate between what was original and what I've added.

**Review by KyleEvers — APPROVED** [review]
Looks good.

File: gather-domains.sh [hunk 1]
```diff
@@ -45,6 +45,10 @@ sed -i '/[^,]*,[^,]*,U\.S\. Courts,/d;/[^,]*,[^,]*,The Supreme Court,/d' $OUTPUT
 ###
 sed -i '/[^,]*,[^,]*,Library of Congress,/d;/[^,]*,[^,]*,Government Publishing Office,/d;/[^,]*,[^,]*,Congressional Office of Compliance,/d;/[^,]*,[^,]*,Stennis Center for Public Service,/d;/[^,]*,[^,]*,U.S. Capitol Police,/d;/[^,]*,[^,]*,Architect of the Capitol,/d' $OUTPUT_DIR/current-federal_modified.csv
 ###
+# Remove DHS
+###
+sed -i '/[^,]*,[^,]*,Department of Homeland Security,/d' $OUTPUT_DIR/current-federal_modified.csv
```
Comment (konklone) [inline gather-domains.sh:50]: Just curious - why drop DHS?

Comment (h-m-f-t) [reply gather-domains.sh:50]: This is a hurricane-related reprieve, and is temporary. They'll be back soon.

**Pull request merged**